Game/player.tscn

```diff
@@ -402,4 +583,7 @@ wait_time = 0.1
 [node name="dash_cooldown_timer" type="Timer" parent="."]
 wait_time = 0.6
 
+[node name="VisibleOnScreenNotifier2D" type="VisibleOnScreenNotifier2D" parent="."]
+
 [connection signal="timeout" from="dash_cooldown_timer" to="." method="_on_dash_cooldown_timer_timeout"]
+[connection signal="screen_exited" from="VisibleOnScreenNotifier2D" to="." method="_on_visible_on_screen_notifier_2d_screen_exited"]
```

Comment:
Please remove the `VisibleOnScreenNotifier2D` node, I think that will also auto-disconnect the signal here.

Reply:
You are saying just to remove line 586 ?